### PR TITLE
Translate "Ruby 3.2.0 Preview 1 Released" to ko

### PR DESCRIPTION
**Pull request by riseshia**
:link: #1894

Actual diff is 915d85c77757d95c0016db42f936d5

Please review this @ruby/www-ruby-lang-org-i18n-ko

### CODE REVIEW

File: ko/news/_posts/2022-04-03-ruby-3-2-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+WASI에 기반해 웹어셈블리를 지원하는 첫 이식판입니다. 이를 통해 CRuby 바이너리는 웹 브라우저, 서버리스 엣지 환경, 그 이외의 웹어셈블리/WASI를 사용가능한 환경에서 동작할 수 있습니다. 현재 이 이식판은 스레드 API이외의 기본적인 테스트와 부트스트랩 테스트 스위트를 통과합니다.
+
+![](https://i.imgur.com/opCgKy2.png)
+
+### 배경
+
+[웹어셈블리 (WASM)](https://webassembly.org/)는 본래 웹 브라우저에서 프로그램을 안전하고 빠르게 실행하기 위해서 만들어졌습니다. 하지만 그 목적 중 하나인 프로그램을 다양한 환경에서 안전하고 효율적으로 실행하는 것은 웹 뿐만이 아니라 일반적인 애플리케이션도 바라던 것입니다.
+
+[WASI (The WebAssembly System Interface)](https://wasi.dev/) is designed for such use cases. Though such applications need to communicate with operating systems, WebAssembly runs on a virtual machine which didn't have a system interface. WASI standardizes it.
```
Comment (marocchino) [inline ko/news/_posts/2022-04-03-ruby-3-2-0-preview1-released.md:24]:
```suggestion
```

File: ko/news/_posts/2022-04-03-ruby-3-2-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+
+### 관련 링크
+
+* [Add WASI based WebAssembly support #5407](https://github.com/ruby/ruby/pull/5407)
+* [An Update on WebAssembly/WASI Support in Ruby](https://itnext.io/final-report-webassembly-wasi-support-in-ruby-4aface7d90c9)
+
+## 정규표현식 타임아웃
+
+정규표현식 일치 처리 중에 타임아웃이 가능해졌습니다.
```
Comment (marocchino) [inline ko/news/_posts/2022-04-03-ruby-3-2-0-preview1-released.md:47]: maybe) 그냥 원문도 나쁘지 않아보이는데요.
```suggestion
정규표현식 일치 처리에 타임아웃 기능을 도입했습니다.
```

**Review by marocchino — APPROVED** [review]

File: ko/news/_posts/2022-04-03-ruby-3-2-0-preview1-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2022-04-03-ruby-3-2-0-preview1-released.md:24]: 🙈

Comment (riseshia) [reply ko/news/_posts/2022-04-03-ruby-3-2-0-preview1-released.md]: fixed 635a4b59

**Comment by riseshia** [comment]
Thanks 🙏

**Pull request merged**